corecomponent/androidcomponent/src/main/res/drawable/bg_bottom_sheet_dialog_fragment.xml

```diff
@@ -8,5 +8,5 @@
         android:topRightRadius="@dimen/bottom_sheet_corner_radius"
         />
     <padding android:top="@dimen/bottom_sheet_top_padding"/>
-    <solid android:color="@color/white"/>
+    <solid android:color="@color/colorSurface"/>
```

Comment:
`white` is classified to these 4 colors.

- `colorBackground` window background
- `colorSurface` surface background (like CardView, BottomSheet)
- `colorOnBrand` text color on brand color
- `colorOnPrimary` text color on primary color (brand and primary should be distinguished for dark mode)

To prepare dark mode, I replaced `@color/white` to each new variables.